contrib/nydusify/cmd/nydusify.go

```diff
@@ -364,7 +364,12 @@ func main() {
 					Usage:   "Read prefetch list from STDIN, please input absolute paths line by line",
 					EnvVars: []string{"PREFETCH_PATTERNS"},
 				},
-
+				&cli.StringFlag{
+					Name:    "compressor",
+					Value:   "lz4_block",
```

Comment:
This default value of `nydus-image` has been changed to `zstd` in the master branch.

Reply:
fixed